index.js

```diff
@@ -1,493 +1,336 @@
+                    else {
+                        return this.camelCase(`delete-${this._singularizedTableName(table)}-by-${detailedKeys
+                            .map((key) => this.column(key))
+                            .join("-and-")}`);
+                    }
+                },
+                updateByKeysInputType(detailedKeys, table, constraint) {
+                    if (constraint.tags.updateFieldName) {
+                        return this.upperCamelCase(`${constraint.tags.updateFieldName}-input`);
+                    }
+                    if (constraint.type === "p") {
+                        // Primary key, shorten!
+                        return this.upperCamelCase(`update-${this._singularizedTableName(table)}-input`);
+                    }
+                    else {
+                        return this.upperCamelCase(`update-${this._singularizedTableName(table)}-by-${detailedKeys
+                            .map((key) => this.column(key))
+                            .join("-and-")}-input`);
+                    }
+                },
+                deleteByKeysInputType(detailedKeys, table, constraint) {
+                    if (constraint.tags.deleteFieldName) {
+                        return this.upperCamelCase(`${constraint.tags.deleteFieldName}-input`);
+                    }
+                    if (constraint.type === "p") {
+                        // Primary key, shorten!
+                        return this.upperCamelCase(`delete-${this._singularizedTableName(table)}-input`);
+                    }
+                    else {
+                        return this.upperCamelCase(`delete-${this._singularizedTableName(table)}-by-${detailedKeys
+                            .map((key) => this.column(key))
+                            .join("-and-")}-input`);
+                    }
+                },
+                updateNode(table) {
+                    return this.camelCase(`update-${this._singularizedTableName(table)}-by-${nodeIdFieldName}`);
+                },
+                deleteNode(table) {
+                    return this.camelCase(`delete-${this._singularizedTableName(table)}-by-${nodeIdFieldName}`);
+                },
+                updateNodeInputType(table) {
+                    return this.upperCamelCase(`update-${this._singularizedTableName(table)}-by-${nodeIdFieldName}-input`);
+                },
+                deleteNodeInputType(table) {
+                    return this.upperCamelCase(`delete-${this._singularizedTableName(table)}-by-${nodeIdFieldName}-input`);
+                },
+            }
+            : null));
+    });
 }
-
+// Hack to set the name in declaration emit
+const PgSimplifyInflectorPlugin = SimplifyInflectorPlugin;
+// Hack for CJS import
 module.exports = PgSimplifyInflectorPlugin;
-// Hacks for TypeScript/Babel import
-module.exports.default = PgSimplifyInflectorPlugin;
-Object.defineProperty(module.exports, "__esModule", { value: true });
```

Comment:
Though TypeScript has added this at the top of the file; it's then overwritten by the `module.exports =` above. So we should add it back, or find a better way.

Reply:
Yeah I wasn't thinking and missed the implication of moving that up.